.github/workflows/build#vsix#publish_to_marketplaces.yml

```diff
@@ -0,0 +1,51 @@
+# Build and publish VSIX bundle of the VScode extension to marketplaces
+name: Publish VSIX to Marketplaces
+
+on:
+  workflow_call:
+    inputs:
+      git_ref:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+
+env:
+  EXTENSION_DIRECTORY: ./vscode-ext
+  
+jobs:
+  publish-vsix:
+    runs-on: ubuntu-latest
+   
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.git_ref }}
+  
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+    
+      - name: Install NPM dependencies
+        working-directory: ${{ env.EXTENSION_DIRECTORY }}
+        run: npm install
+        
+      - name: Package the extension
+        working-directory: ${{ env.EXTENSION_DIRECTORY }}
+        run: npm run package
+
+      - name: Align package filename with release version
+        working-directory: ${{ env.EXTENSION_DIRECTORY }}
+        run: |
+          PACKAGE_VERSION="$(
+            cat package.json | jq '.version' --raw-output
+          )"
+          mv caesar-${PACKAGE_VERSION}.vsix caesar-${{ inputs.version }}.vsix
```

Comment:
Can you explain why you move the package here? Shouldn’t the version always agree with the tag?